docs/platform/release-notes/tizen-2-4.md

```diff
@@ -204,7 +202,6 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 
   - Ubuntu sometimes stops responding for a few seconds after closing the Emulator Manager. This issue is related to a bug of Intelligent Input Bus (IBus). When the issue occurs, restart the ibus-daemon by entering the ibus-daemon –drx command at the command prompt, and use another framework, such as uim and fcitx, for multilingual input.
   - To use the Tizen Emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver which the vendor provides. Check the prerequisites for the Tizen Emulator from:
-    - [https://developer.tizen.org/downloads/sdk/installing-sdk/prerequisites-tizen-sdk](https://developer.tizen.org/downloads/sdk/installing-sdk/prerequisites-tizen-sdk).
```

Comment:
How about removing "Check the prerequisites for the Tizen Emulator from:" of above line also?